examples/src/main/python/ml/binarizer_example.py

```diff
@@ -17,9 +17,10 @@
 
 from __future__ import print_function
 
-from pyspark.sql import SparkSession
 # $example on$
 from pyspark.ml.feature import Binarizer
+from pyspark.sql import SparkSession
```

Comment:
So we might want to move the $example off$ tag/comment up above this so that we keep the example text the same.